backend/src/data/repositories/episode/episode.repository.ts

```diff
@@ -112,13 +112,13 @@ class Episode {
       .resultSize();
   }
 
-  public getAllByPLaylistId(isOwner: boolean, playlistId: number): Promise<TEpisodeWithPodcast[]> {
+  public getAllByPLaylistId(playlistId: number): Promise<TEpisodeWithPodcast[]> {
     return this.#EpisodeModel
       .query()
       .withGraphJoined('[podcast]')
       .joinRelated('[playlistEpisodes]')
       .where('playlistEpisodes.playlist_id', playlistId)
-      .where((builder) => isOwner || builder.where('episodes.type', EpisodeType.PUBLIC));
+      .where((builder) => builder.where('episodes.type', EpisodeType.PUBLIC));
```

Comment:
From this point, private episodes become unreachable for everyone, including the owner.

Reply:
agreed
owner need to see all episodes